specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/mongorbac.json

```diff
@@ -406,7 +406,7 @@
           "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
         },
         "type": {
-          "type": "string",
+          "type": "int",
```

Comment:
It should be "integer" not "int"?

Reply:
It should be `integer` instead of `int` as far as I know

Reply:
Done